release_scripts/broad_rgp_release.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-gcloud storage --project=broad-rgp cp --recursive \
+gcloud storage --billing-project broad-rgp cp --recursive \
```

Comment:
In case this doesn't work, you might have to move the `--billing-project` before `storage`.